### PR TITLE
Song book updates

### PR DESCRIPTION
**Pull request by Edde333**
Number of views on songs is now only visible to admin accounts. One can now search on category and tests_data.rake now contains the creation of two songs. Menu for Song Book also added to seeds.

### CODE REVIEW

**Comment by Edde333** [comment]
partly solves issue #630

File: lib/tasks/tests_data.rake [hunk 1]
```diff
@@ -151,5 +151,16 @@ namespace :db do
 
     group = Group.find_or_create_by!(group_type: Group::REGULAR, name: 'Black Knight', number: 1, introduction: introduction)
     GroupUser.find_or_create_by!(group: group, user: user, fadder: true)
+
+    # Songs
+    Song.find_or_create_by!(title: 'Hello World',
+                                    author: 'Edwin',
+                                    category: 'Absurda Visor',
+                                    content: 'Det var en gång en liten Värld som utplånades! Skål!')
```
Comment (olofenglund96) [inline lib/tasks/tests_data.rake]: Probably align these attributes like the others just below

File: app/views/songs/index.html.erb [hunk 2]
```diff
@@ -8,10 +8,12 @@
       link_to(song, song_path(song))
     end
     g.column(name: Song.human_attribute_name(:author), attribute: 'author')
-    g.column(name: Song.human_attribute_name(:category), attribute: 'category', filter: false)
-    g.column(name: Song.human_attribute_name(:visits), attribute: 'visits', filter: false) do |song|
-    end
+    g.column(name: Song.human_attribute_name(:category), attribute: 'category')
+
+      
+
```
Comment (olofenglund96) [inline app/views/songs/index.html.erb]: Remove these whitelines plssss

**Comment by navro** [comment]
Nice work!! :smile: 
Just fix @olofenglund96's comments, then merge!

**Pull request merged**